### PR TITLE
Update the reference to the default branch

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

**Comment by mdellweg** [comment]
The change looks good. Is it possible, the fixtures changed and that checksum ceased to exist?

**Comment by lubosmj** [comment]
@mdellweg, yes, exactly. The fixtures were re-generated a couple of days ago. As of that moment, the CI is failing all the time.

**Comment by mdellweg** [comment]
If you like, we can admin merge this change anyway.

I'd suggest to rework the tests to not hardcode the checksum.

File: tests/scripts/pulp_ostree/test_modify_content.sh [hunk 1]
```diff
@@ -16,35 +16,40 @@ expect_succ pulp ostree repository create --name "cli_test_ostree_repository1"
 expect_succ pulp ostree repository sync --name "cli_test_ostree_repository1" \
   --remote "cli_test_ostree_remote"
 
+expect_succ pulp ostree repository content list --repository cli_test_ostree_repository1 --type ref
+COMMIT_CHECKSUM1=$(echo "$OUTPUT" | jq -r ".[0].checksum")
+COMMIT_CHECKSUM2=$(echo "$OUTPUT" | jq -r ".[1].checksum")
+REF_NAME2=$(echo "OUTPUT" | jq -e ".[1]".checksum)
```
Comment (mdellweg) [inline tests/scripts/pulp_ostree/test_modify_content.sh:22]: Single quotes are usually the easier escaping for jq statements.
```suggestion
COMMIT_CHECKSUM1=$(echo "$OUTPUT" | jq -r '.[0].checksum')
COMMIT_CHECKSUM2=$(echo "$OUTPUT" | jq -r '.[1].checksum')
REF_NAME2=$(echo "$OUTPUT" | jq -r '.[1].name')
```

**Pull request merged**